src/openrct2/interface/Widget.h

```diff
@@ -13,43 +13,16 @@
 #include "../localisation/StringIds.h"
 #include "Window.h"
 
-enum WINDOW_WIDGET_TYPES
-{
-    WWT_EMPTY = 0,
-    WWT_FRAME = 1,
-    WWT_RESIZE = 2,
-    WWT_IMGBTN = 3,
-    WWT_COLOURBTN = 6,
-    WWT_TRNBTN = 7,
-    WWT_TAB = 8,
-    WWT_FLATBTN = 9,
-    WWT_BUTTON = 10,
-    WWT_LABEL_CENTRED = 12, // Centred text
-    WWT_TABLE_HEADER = 13,  // Left-aligned textual button
-    WWT_LABEL = 14,         // Left-aligned text
-    WWT_SPINNER = 15,
-    WWT_DROPDOWN = 16,
-    WWT_VIEWPORT = 17,
-    WWT_GROUPBOX = 19,
-    WWT_CAPTION = 20,
-    WWT_CLOSEBOX = 21,
-    WWT_SCROLL = 22,
-    WWT_CHECKBOX = 23,
-    WWT_PLACEHOLDER = 25,
-    WWT_TEXT_BOX = 27,
-    WWT_LAST = 26,
```

Comment:
Please restore the explicit number assignment for now. I know it is done automatically, but if you take a close look we skip some number like `4, 5, 11, 18, 24` and I don't know currently if that will do any harm.